panopticum/management/commands/importer.py

```diff
@@ -0,0 +1,1085 @@
+            # dbobj = obj[0]
+            # new_obj = obj[1]
+            if dbobj is None:
+                pass  # TODO: Exception
+
+    @staticmethod
+    def get_comp_runtimetypes(name, order=1):  # Creates record in DB if doesn't exists.
+        return Utility.alloc_obj_by_name(ComponentRuntimeTypeModel, name, order=order)
+
+    @staticmethod
+    def seed_comp_runtimetypes():
+        # fn_name = 'Utility::seed_comp_runtimetypes() -'
+        logging.info('starts')
+        for name in Utility.comp_runtimes:
+            dbobj = Utility.get_comp_runtimetypes(name, 1)
+            if dbobj is None:
+                pass  # TODO: Exception
+
+    @staticmethod
+    def get_country(name):
+        return Utility.alloc_obj_by_name(CountryModel, name)
+
+    @staticmethod
+    def seed_countries():
+        # fn_name = 'Utility::seed_countries() -'
+        logging.info('starts')
+
+        for name in Utility.countries:
+            dbobj = Utility.get_country(name)
+            if dbobj is None:
+                pass  # TODO: Exception
+
+    @staticmethod
+    def get_datacenter(name, info, grafana, metrics):
+        return Utility.alloc_obj_by_name(DatacenterModel, name, info=info, grafana=grafana, metrics=metrics)
+
+    @staticmethod
+    def seed_datacenters():  # TODO: Need to consider components_deployments field (M-M) data - we don't have seed
+        # data yet.
+        # fn_name = 'Utility::seed_datacenters() -'
+        logging.info('starts')
+        for (dc_name, dc_values) in Utility.data_centers.items():
+            dbobj = Utility.get_datacenter(dc_name, dc_values['info'], dc_values['grafana'], dc_values['metrics'])
+            if dbobj is None:
+                pass  # TODO: Exception
+            elif dbobj.grafana != dc_values['grafana'] or dbobj.metrics != dc_values['metrics'] or dbobj.info != \
+                    dc_values['info']:
+                logging.info(
+                    'Ignoring seed value! [%s] exists in DB with different grafana or metrics value. Seed '
+                    'info value [%s], grafana value [%s], metrics value [%s].',
+                     dc_name, dc_values['info'], dc_values['grafana'], dc_values['metrics'])
+
+    @staticmethod
+    def get_db_vendor(name):
+        return Utility.alloc_obj_by_name(DatabaseVendorModel, name)
+
+    @staticmethod
+    def seed_db_vendors():
+        # fn_name = 'Utility::seed_db_vendors() -'
+        logging.info('starts')
+
+        for name in Utility.db_vendors:
+            dbobj = Utility.get_db_vendor(name)
+            # dbobj = obj[0]
+            # new_obj= obj[1]
+            if dbobj is None:
+                pass  # TODO: Exception
+
+    @staticmethod
+    def get_depl_env(name):
+        return Utility.alloc_obj_by_name(DeploymentEnvironmentModel, name)
+
+    @staticmethod
+    def seed_depl_envs():
+        # fn_name = 'Utility::seed_depl_envs() -'
+        logging.info('starts')

```

Comment:
I see such logging.info('starts') pattern repeats everywhere and not informative, probably it's better to add good logging directly to alloc_obj_by_name()

Reply:
Noted.